src/types/crossChainAction.ts

```diff
@@ -46,6 +46,7 @@ interface KlimaStakingActionPreview {
 }
 
 interface PlrStakingActionPreview {
+  isUnStake?: boolean | null,
```

Comment:
I think you can remove the `| null`, there shouldn't really be a situation where someone would pass null through this param, it will either be a boolean or undefined